src/main/java/edu/project2/tripplanner/exception/NotFoundExceptions.java

```diff
@@ -0,0 +1,8 @@
+package edu.project2.tripplanner.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class NotFoundExceptions extends RuntimeException {
```

Comment:
class name can't be plural
exception, no exception.
Also, please add some additional info. What is not found?